R/teal_slice.R

```diff
@@ -483,44 +530,51 @@ c.teal_slices <- function(...) {
   )
 }
 
+# store method for `teal_slices` object
+#' @param x `teal_slices` object
+#' @param file a `character(1)` with the storage destination name
+#' @export
+#' @rdname teal_slice
+#' @keywords internal
+#'
+store.teal_slices <- function(x, file = NULL, ...) {
```

Comment:
Why would NULL be acceptable?

Reply:
agh, an artifact from a previous idea. thanks! no needed here